influence_benchmark/root.py

```diff
@@ -1,5 +1,9 @@
 from pathlib import Path
 
+from dotenv import load_dotenv
```

Comment:
does this mean that only code which at some point uses a path in this file will have access to api keys?

Reply:
Yes, good point. Hasn't seemed to be an issue for the code I've used so far.